extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/NarayanaQuarkusConfiguration.java

```diff
@@ -0,0 +1,147 @@
+package io.quarkus.narayana.jta.runtime;
+
+import java.util.List;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+public class NarayanaQuarkusConfiguration {
+    // non final properties
+    /**
+     * config
+     */
+    @ConfigItem(name = "CoordinatorEnvironmentBean.commitOnePhase", defaultValue = "true")
+    public boolean commitOnePhase;
+    /**
+     * config
+     */
+    @ConfigItem(name = "ObjectStoreEnvironmentBean.objectStoreDir", defaultValue = "PutObjectStoreDirHere")
+    public String objectStoreDir;
+    /**
+     * config
+     */
+    @ConfigItem(name = "ObjectStoreEnvironmentBean.transactionSync", defaultValue = "true")
+    public boolean transactionSync;
+
+    //    @ConfigItem(name = "CoreEnvironmentBean.nodeIdentifier", defaultValue = "1")
+    //    public String nodeIdentifier;
+    /**
+     * config
+     */
+    @ConfigItem(name = "JTAEnvironmentBean.xaRecoveryNodes", defaultValue = "1")
```

Comment:
I think there should not be configurable to change the `xaRecoveryNodes`. The xa recovery nodes should be changed to what is set in node name. Having the node name and recovery nodes to be the same value is safer as it diminishes possibility of misconfiguration.
See `NarayanaJtaRecorder.setNodeName` method.

It's a way how WFLY integration works.

Reply:
Okay. The initial PR duplicates the default properties bundled in with the Narayana jar.
Should we still set the property but not allow the user to reconfigure it.